spec/slack/web/client_spec.rb

```diff
@@ -150,6 +150,47 @@
       end
     end
 
+    context 'adapter option' do
+      let(:adapter) { Faraday.default_adapter }
+      let(:adapter_class) { Faraday::Adapter::NetHttp }
+
+      before do
```

Comment:
You need to reset the default adapter to the old value around the spec to avoid polluting other spec results.